### PR TITLE
fix flickering

### PR DESCRIPTION
**Pull request by Overman775**
Fix:

- Text flickering,
- Black screen,
- double rebuild main widget 

@aissat  @moritzmorgenroth Please look my idea, not for release :))

### CODE REVIEW

**Comment by aissat** [comment]
nice working well

**Comment by Overman775** [comment]
> need fix tests

Yep 👍

File: lib/src/easy_localization_app.dart [hunk 1]
```diff
@@ -99,6 +101,16 @@ class EasyLocalizationProvider extends InheritedWidget {
...
 
+  @override
+  Widget get child {
+    if(_locale == null){
+      return PreloaderWidget(null, parent.preloaderColor);
+    }
+    else{
+      return PreloaderWidget(super.child, parent.preloaderColor);
+    } 
+  }
```
Comment (aissat) [inline lib/src/easy_localization_app.dart]: thinking this part like `FutureBuilder`

**Comment by aissat** [comment]
> > need fix tests
> 
> Yep 👍
@Overman775 
thinking failed test coz something wrong in ur test, i have other bug when run app blocked in PreloaderWidget

**Comment by Overman775** [comment]
@aissat fixed, checked, and part of test works `+40 -5: Some tests failed.`
i don't have experience in flutter tests...

@aissat 
The main idea is to catch widget.child and replace it with a simple container, this will delete text flicker and improve performance. 
Then put widget.child into a color container to avoid black screen flickering

**Pull request closed without merging**